src/ChainSafe.Gaming.WalletConnect/WalletConnectSigner.cs

```diff
@@ -92,12 +85,12 @@ public async ValueTask WillStartAsync()
             }
 
             // Wallet Connect
-            await Initialize(configuration.WalletConnectConfig);
+            await Initialize(configuration);
```

Comment:
This can be inlined back to the WillStartAsync as there is only WalletConnect now.

Reply:
what do you mean, please elaborate

Reply:
done, I've refactored the signer to use only one config reference that's injected and readonly